### PR TITLE
[BEAM-5485] Fix time unit discrepancies in windowing documentation

### PR DESCRIPTION
**Pull request by deepyaman**
Fix time unit discrepancies windowing documentation so that Python snippets match descriptions and Java code

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Flink/lastCompletedBuild/) | --- | --- | ---

### CODE REVIEW

**Comment by deepyaman** [comment]
@jasonkuster Going to @ you, since you show assigned to the JIRA issue (for whatever reason). Not sure why checks are failing... I changed minutes to seconds in a few locations, and since they're the same length it can't even be a style violation. Details aren't giving me any clues.

**Comment by jasonkuster** [comment]
@alanmyrvold @yifanzou @markflyhigh may be able to better assist here.

File: website/src/documentation/programming-guide.md [hunk 1]
```diff
@@ -2253,13 +2253,13 @@ into fixed windows, each one minute in length:
 #### 7.3.2. Sliding time windows {#using-sliding-time-windows}
 
 The following example code shows how to apply `Window` to divide a `PCollection`
-into sliding time windows. Each window is 30 minutes in length, and a new window
+into sliding time windows. Each window is 30 seconds in length, and a new window
```
Comment (yifanzou) [inline website/src/documentation/programming-guide.md]: I don't think we fully migrate the website into the beam code base yet. We might still need to update this in the beam-site. @alanmyrvold correct me if I am wrong.

Comment (melap) [reply website/src/documentation/programming-guide.md]: Yes, the website migration is not finished so website changes need to be made in the beam-site repo's src directory for now. https://github.com/apache/beam-site

Comment (deepyaman) [reply website/src/documentation/programming-guide.md]: Done! apache/beam-site#559

**Comment by deepyaman** [comment]
I still don't understand why checks are failing. I saw this morning that there were no test failures in the Python build, so I tried triggering the checks again to see if it was a one-time issue, but they're still failing. @yifanzou

**Comment by melap** [comment]
I don't know about the Python tests, but if you remove your changes to the website programming guide file from the PR, that should stop the Website precommit failures.

**Comment by melap** [comment]
R: @aaltay

**Comment by yifanzou** [comment]
Run Python PreCommit

**Comment by deepyaman** [comment]
@yifanzou Passes Python PreCommit after switching to master. Since changes to site were moved out to apache/beam-site, merging to master should be good.

**Comment by yifanzou** [comment]
LGTM. @jasonkuster could you merge this PR?

**Comment by jasonkuster** [comment]
LGTM, merging.

**Pull request merged**